src/pages/docs/tutorials/tutorial-kitten-browser/app-setup/index.md

```diff
@@ -18,104 +18,105 @@ the guide.
 > If you're not a fan of kittens, you're welcome to substitute a different keyword in the URLs.  No judgments.
 
 ### Directory Structure
-
-	+ App
-		+ src						<-- All of our source code
-			+ App					<-- The App component to be rendered into the DOM
-				- App.js
-				- package.json
-			+ components			<-- Any reusable components for our App
-				+ Kitten
-					Kitten.js
-					Kitten.less
-					package.json
-			+ views					<-- Composite components that make up a distinct view of the app
-				Detail.js
-				List.js
-			- index.js				<-- The entry point for the module
-		- package.json				<-- Module meta-data
-
+```
++ App
+	+ src						<-- All of our source code
+		+ App					<-- The App component to be rendered into the DOM
+			- App.js
+			- package.json
+		+ components			<-- Any reusable components for our App
+			+ Kitten
+				Kitten.js
+				Kitten.less
+				package.json
+		+ views					<-- Composite components that make up a distinct view of the app
+			Detail.js
+			List.js
+		- index.js				<-- The entry point for the module
+	- package.json				<-- Module meta-data
+```
 ### ./package.json
 
 Let's give our module a name and establish its dependencies.  Edit `package.json` and update the `name` property.
 
-	{
-	  "name": "enact-tutorial-kitten-browser",
-	  "version": "1.0.0",
-      "description": "A general template for an Enact Moonstone application.",
-      "author": "",
-      "main": "src/index.js",
-      "scripts": { [omitted] },
-      "license": "UNLICENSED",
-      "private": true,	
-      "repository": "",
-      "enact": {
-          "theme": "moonstone"
-      },
-      "eslintConfig": {
-          "extends": "enact"
-      },
-      "dependencies": { [omitted] }
-	}
-
+```json
+{
+	"name": "enact-tutorial-kitten-browser",
+	"version": "1.0.0",
+	"description": "A general template for an Enact Moonstone application.",
+	"author": "",
+	"main": "src/index.js",
+	"scripts": { [omitted] },
+	"license": "UNLICENSED",
+	"private": true,	
+	"repository": "",
+	"enact": {
+		"theme": "moonstone"
+	},
+	"eslintConfig": {
+		"extends": "enact"
+	},
+	"dependencies": { [omitted] }
+}
+```
 ### ./src/index.js
+```js
+import React from 'react';
+import {render} from 'react-dom';
 
-	import React from 'react';
-	import {render} from 'react-dom';
-	
-	import App from './App';
-	
-	let appElement = (<App />);
-	
-	// In a browser environment, render instead of exporting
-	if (typeof window !== 'undefined') {
-		render(
-			appElement,
-			document.getElementById('root') // provided by Enact's HTML template
-		);
-	}
-	
-	export default appElement;
+import App from './App';
 
-## App
+let appElement = (<App />);
 
-### ./src/App/package.json
+// In a browser environment, render instead of exporting
```

Comment:
I think a close and open set of code block are missing here and above on 67 & 69.

Reply:
67 to 68 are already close and open set of code block.

Reply:
Oh, I see.  the diff view confused me with the headers.   :disappear: